packages/react/src/components/ComposedModal/ComposedModal-story.js

```diff
@@ -28,14 +35,17 @@ const sizes = {
 };
 
 const buttons = {
-  'None (0)': 0,
-  'One (1)': 1,
-  'Two (2)': 2,
+  'None (0)': '0',
```

Comment:
Seems like `0` doesn't do anything unless we specify the `passiveModal` prop, so should we just allow choosing 1, 2, or 3 buttons, and controlling the 0 option via the `passiveModal` knob?